src/software/ai/navigator/path_planner/global_path_planner_factory.cpp

```diff
@@ -52,10 +52,9 @@ GlobalPathPlannerFactory::GlobalPathPlannerFactory(
         {
             auto obstacles = obstacle_factory.createStaticObstaclesFromMotionConstraints(
                 static_motion_constraints, field);
-            planners.emplace(std::make_pair(
-                static_motion_constraints,
-                std::make_shared<EnlsvgPathPlanner>(field.fieldBoundary(), obstacles,
-                                                    ROBOT_MAX_RADIUS_METERS)));
+            planners.emplace(std::make_pair(static_motion_constraints,
+                                            std::make_shared<EnlsvgPathPlanner>(
+                                                field.fieldBoundary(), obstacles, 0)));
```

Comment:
if you make this change, can you add AVOID_FIELD_BOUNDARY_ZONE to this file https://github.com/UBC-Thunderbots/Software/blob/8607d4565479f032a11333eca817026bca9b8873/src/software/ai/motion_constraint/motion_constraint_set_builder.cpp#L27 and override it for the BallPlacementTactic in the motion_constraint_visitor